modules/base/tool.go

```diff
@@ -41,6 +41,14 @@ func ShortSha(sha1 string) string {
 	return TruncateString(sha1, 10)
 }
 
+// ShortSha256 takes 7-17 indexes of sha256 string as short sha256
+func ShortSha256(sha256 string) string {
+	if utf8.RuneCountInString(sha256) < 17 {
+		return sha256
+	}
+	return string([]rune(sha256)[7:17])
+}
+
```

Comment:
I think we might already have such a function for commit SHA's, could reuse it.

Reply:
`ShortSha`?

Reply:
Yes, that one.

Reply:
fixed